ocaml/typing/printtyp.ml

```diff
@@ -3274,36 +3108,9 @@ let explain mis ppf =
 let warn_on_missing_def env ppf t =
   match get_desc t with
   | Tconstr (p,_,_) ->
-<<<<<<< HEAD
     begin match Env.find_type p env with
-    | { type_kind = Type_abstract Abstract_rec_check_regularity; _ } ->
-||||||| 121bedcfd2
-    begin
-      try
-        ignore(Env.find_type p env : Types.type_declaration)
-      with Not_found ->
-=======
-    begin match Env.find_type p env with
-    | exception Not_found ->
->>>>>>> 5.2.0
-        fprintf ppf
-<<<<<<< HEAD
-          "@,@[<hov>Type %a was considered abstract@ when checking\
```

Comment:
It looks like we're losing this, but I don't think we should.

Reply:
I think we retain it below: https://github.com/ocaml-flambda/flambda-backend/pull/2821/files#diff-384fa66d4466a5b8c1e3dd8883e02c0c905aa4377e84be7c5afad25d0aa93cd5L3310-L3317